### PR TITLE
jormungandr: address added in poi

### PR DESCRIPTION
**Pull request by kadhikari**
Address added in poi in the result autocomplete bragi.

Concerned ticket: https://jira.canaltp.fr/browse/NAVP-721

### CODE REVIEW

File: source/jormungandr/jormungandr/autocomplete/geocodejson.py [hunk 1]
```diff
@@ -42,6 +42,14 @@
 import re
 
 
+def get_number(housenumber):
+    hn = 0
+    numbers = re.findall(r'^\d+', housenumber or "0")
```
Comment (xlqian) [inline source/jormungandr/jormungandr/autocomplete/geocodejson.py]: repeatedly used code, you can put it utils.py

File: source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py [hunk 2]
```diff
@@ -152,6 +152,7 @@ class PoiSerializer(serpy.DictSerializer):
     administrative_regions = AdministrativeRegionsSerializer()
     poi_type = jsonschema.MethodField(display_none=False)
     properties = jsonschema.MethodField(display_none=False)
+    address = jsonschema.MethodField(display_none=False)
```
Comment (xlqian) [inline source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py:157]: we don't have a AddressSerializer?

Comment (kadhikari) [reply source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py:157]: Yes we have but they read attributs directly in "properties.geocoding"

File: source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py [hunk 3]
```diff
@@ -161,6 +163,22 @@ def get_properties(self, obj):
         return {p.get("key"): p.get("value") 
                 for p in obj.get('properties', {}).get('geocoding', {}).get('properties', [])}
 
+    def get_address(self, obj):
+        address = obj.get('properties', {}).get('geocoding', {}).get('address', None)
+        if not address:
+            return None
+
+        return {
```
Comment (pbougue) [inline source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py]: This is duplicated it seems (close from https://github.com/CanalTP/navitia/pull/2300/files#diff-204c59b4be2984c009079df1061d66f7R131and maybe even from https://github.com/CanalTP/navitia/pull/2300/files#diff-204c59b4be2984c009079df1061d66f7R180 )

**Review by xlqian — APPROVED** [review]

**Comment by benoit-bst** [comment]
One question  : Is there a test when no adress has returned ?

**Review by benoit-bst — APPROVED** [review]

**Review by pbougue — APPROVED** [review]

**Pull request merged**